Core/HLE/scePsmf.cpp

```diff
@@ -1229,6 +1229,10 @@ static int _PsmfPlayerSetPsmfOffset(u32 psmfPlayer, const char *filename, int of
 	psmfplayer->videoWidth = buf[142] * 16;
 	psmfplayer->videoHeight = buf[143] * 16;
 
+	// Reflect these values to memory - LocoRoco2 reads the height directly here.
+	Memory::Write_U32(psmfplayer->videoWidth, psmfPlayer + 0xBC);
```

Comment:
Tested a few lib versions quickly - `scePsmfPlayerCreate()` takes away 290816 (0x47000) bytes of RAM.  It allocates low (like an FPL without PSP_FPL_ATTR_HIGHMEM.)

Interestingly, it *seems* like the pointer written by scePsmfPlayerCreate is actually 80 (0x50) behind the real allocated memory.  That is, `*p = (PsmfPlayerData *)((intptr_t)allocated - 0x50);`  So the width/height are probably at 0x6C and 0x70 from the actual memory.

That being said, I still end up with 0 at player + 0xbc and 0xc0, it seems like, even after calling `scePsmfPlayerGetPsmfInfo()`.  But probably I'm just missing something.

-[Unknown]